uiowa/uiowa
# [Facilities] Added fixes for project page and views

Resolves https://github.com/uiowa/uiowa/issues/7526

# How to test

```
ddev blt ds --site=facilities.uiowa.edu
```
1. Code review or
2. Go to https://facilities.uiowa.ddev.site/project-information and confirm form is using ajax and search field is also searching for project_building_alt field. 
3. Go to https://facilities.uiowa.ddev.site/project-information/featured-projects and confirm building name is listed in the table and sorting is done through ajax. 
4. Go to https://facilities.uiowa.ddev.site/projects/project-information/capital-projects and confirm building name is listed in the table and sorting is done through ajax. 
5. Go to https://facilities.uiowa.ddev.site/project/0963101 and confirm project building is displaying on node.
